scripts/stress-test/docker-compose.yml

```diff
@@ -31,6 +33,14 @@ services:
       - POSTGRES_PASSWORD=postgresrootpassword
       - POSTGRES_USER=postgres
 
+  pgnative:
+    image: mdillon/postgis
```

Comment:
what's the advantage of using this image over vanilla pg?

Reply:
I'm not too sure if there are technical advantages, but the `postgresql` strategy uses it above on line 27. I assumed there was a good reason for it. For benefit of `pg-native` development, it makes debugging easier by ensuring there aren't any odd bugs because of configuration issues across two different images.